windows-vagrant.md

```diff
@@ -294,7 +294,7 @@ rails s -b 0.0.0.0 -p 3000
 
 > **Important**: When the server finishes starting up, it may appear that it is hanging.  The result will look like this:
 >
-> ![server](images/rails-server-running)
+> ![server](images/rails-server-running.png)
```

Comment:
Ideally this could be a screenshot of this running on a windows machine, but for v1, seeing a screenshot in a different OS seems ok for now.